doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -566,6 +566,7 @@ The `mz_sessions` table contains a row for each active session in the system.
 | `id`            | [`uint4`]                      | The ID of the session.                                                                                                    |
 | `role_id`       | [`text`]                       | The role ID of the role that the session is logged in as. Corresponds to [`mz_catalog.mz_roles`](../mz_catalog#mz_roles). |
 | `connected_at`  | [`timestamp with time zone`]   | The time at which the session connected to the system.                                                                    |
+| `uuid`          | [`uuid`]                       | The UUID of the session. Corresponds to [`mz_recent_activity_log.session_id](#mz_recent_activity_log).                    |
```

Comment:
Having an ID and a UUID is confusing! At the very least we should explain with more words what the difference is. Possibly we should also pick more descriptive names for these fields. E.g., perhaps `id` should be called `connection_id`, and `uuid` should be called `id`.

Reply:
+1